README.md

```diff
@@ -31,9 +31,9 @@ tar stream into the `Decoder` which emits "entry" events for each individual fil
 
 require __DIR__ . '/vendor/autoload.php';
 
-$stream = new ReadableResourceStream(fopen('archive.tar', 'r'));
+$stream = new React/Stream/ReadableResourceStream(fopen('archive.tar', 'r'));
```

Comment:
You're currently using the wrong slashes in here, I added a suggested change below. My suggestions also aligns with PSR-4, maybe worth to take a look at https://www.php-fig.org/psr/psr-4/ 

```suggestion
$stream = new React\Stream\ReadableResourceStream(fopen('archive.tar', 'r'));
```